spotlessSelf.gradle

```diff
@@ -24,9 +24,12 @@ spotless {
 		trimTrailingWhitespace()
 	}
 	groovy {
-		target 'build.gradle', 'spotlessSelf.gradle', 'settings.gradle'
-		greclipseFormat().configFile('spotless.eclipseformat.xml')
-		paddedCell()
+		target fileTree('.') {
+			include '**/*.gradle'
+			exclude '_ext/**'
+		}
+		custom 'preventFormatPingPong', { return it.replaceAll('}[ \t\f]+}', '}}') }
```

Comment:
The \t should be sensible, since it will most likely to the ping-pong as well. But yes the \f should be removed.